guardian/grid-feeds
# Pull private application config from parameter store

## What does this change?
This uses the [simple-configuration](https://github.com/guardian/simple-configuration) to pull config from parameter store rather than relying on a local file. This will still use local config if running the app locally in DEV mode.

It also removes the route we've been using for testing (`/start`) and kicks off the associate press service when the app is loaded instead

## How to test
Run this app locally (instructions in the README) and hit `http://localhost:8855/management/healthcheck`. Does the app come up as expected? If you have previously pulled down config, you will need to re-run `./scripts/fetch-config.sh`
